src/CreateScenarioButton/components/CreateScenarioDialog/CreateScenarioDialog.js

```diff
@@ -279,17 +285,22 @@ const CreateScenarioDialog = ({
         </Grid>
       </DialogContent>
       <DialogActions className={classes.dialogActions}>
-          <Button id="cancel"
-                  onClick={handleCloseDialog}
-                  color="primary">
-              {t(CREATE_SCENARIO_LABELS.DIALOG_CANCEL, 'Cancel')}
-          </Button>
-          <Button id="create"
-                  disabled={createScenarioDisabled}
-                  onClick={handleCreateScenario}
-                  color="primary">
-              {t(CREATE_SCENARIO_LABELS.DIALOG_CREATE, 'Create')}
-          </Button>
+        <Button
+          id="cancel"
+          onClick={handleCloseDialog}
+          color="primary"
+        >
+          {t(CREATE_SCENARIO_LABELS.DIALOG_CANCEL, 'Cancel')}
+        </Button>
+        <Button
+          id="create"
+          data-cy="scenario-submit-button"
```

Comment:
Could be interesting to prefix the id by  `create-scenario-dialog` in order to narrow the possibilities to have several field with the same data-cy id?
Something like  `create-scenario-dialog-submit-button`